src/bthread/task_group.cpp

```diff
@@ -781,6 +851,7 @@ void TaskGroup::ready_to_run_remote(bthread_t tid, bool nosignal) {
         _remote_num_nosignal.store(0, std::memory_order_release);
         _remote_nsignaled.fetch_add(additional_signal, std::memory_order_release);
         _control->signal_task(1 + additional_signal);
+//        _control->signal_group(group_id_);
```

Comment:
A pthread puts a bthread task into this task group. Why shouldn't ``signal_group`` be called?